docs/modules/ROOT/pages/how-tos/crossplane_service_broker/overview.adoc

```diff
@@ -0,0 +1,23 @@
+= Overview
```

Comment:
This page is arguably more of an explanation than a how-to, it might make sense to move it to the explanations section of the documentation.

Reply:
It is, but it fits nicely into the sequence of the how-tos, especially if you browse through the how-tos in succession. I will leave it therefore there for now.